pages/platform/kubernetes-k8s/installing-operator/guide.en-gb.md

```diff
@@ -0,0 +1,789 @@
+ => [internal] load .dockerignore                                                                                                           0.0s
+ => => transferring context: 2B                                                                                                             0.0s
+ => [internal] load metadata for quay.io/operator-framework/helm-operator:v1.17.0                                                           0.4s
+ => [internal] load build context                                                                                                           0.0s
+ => => transferring context: 659B                                                                                                           0.0s
+ => [1/4] FROM quay.io/operator-framework/helm-operator:v1.17.0@sha256:516e69e6c0c78cca64fd3fe2a43703cf709f399bcfc36dabbc238f8d90954e59     0.0s
+ => CACHED [2/4] COPY watches.yaml /opt/helm/watches.yaml                                                                                   0.0s
+ => CACHED [3/4] COPY helm-charts  /opt/helm/helm-charts                                                                                    0.0s
+ => CACHED [4/4] WORKDIR /opt/helm                                                                                                          0.0s
+ => exporting to image                                                                                                                      0.0s
+ => => exporting layers                                                                                                                     0.0s
+ => => writing image sha256:0f71fd44ff901ed21977e5079a68d3aade40f0cca4550a72365241b5cf450f98                                                0.0s
+ => => naming to 56hkk1xk.gra7.container-registry.ovh.net/example/ovh-nginx-operator:1.0.0     
+
+$ docker login https://56hkk1xk.gra7.container-registry.ovh.net
+Username: harbor-user1
+Password: 
+
+Login Succeeded
+
+$ docker push 56hkk1xk.gra7.container-registry.ovh.net/example/ovh-nginx-operator:1.0.0
+The push refers to repository [56hkk1xk.gra7.container-registry.ovh.net/example/ovh-nginx-operator]
+5f70bf18a086: Pushed 
+4edd44e1433c: Pushed 
+35217553513b: Pushed 
+7766e4bae829: Pushed 
+895f2ebb55fa: Pushed 
+ede2e4397fdc: Pushed 
+87cd41b1f9f8: Pushed 
+44f62afd0479: Pushed 
+1.0.0: digest: sha256:509549a6bac0a2e52a19b4bbac80b5411a2c0fe581c5fbd2cc6b4a456e339eb3 size: 1984
+</code></pre>
+
+
+The last step consistes to deploy your operator in the Kubernetes cluster.  
+First, create the namespace for the operator:
+```bash
+kubectl create ns ovh-nginx-operator
+```
+Output should be like this:
+<pre class="console"><code>
+$ kubectl create ns ovh-nginx-operator
+
+namespace/ovh-nginx-operator created
+</code></pre>
+
+If you have pushed your image in a private registry, you have to create the secret:
```

Comment:
Now you need to create a Kubernetes secret in order to tell Kubernetes what are the credentials to login and pull images to your Docker private registry: